### PR TITLE
Convert more recipes to RA2

### PR DESCRIPTION
**Pull request by chochem**
- MetaTE recipes
- lens oredict recipes
- rotor oredict recipes
- stone oredict recipes
- gem oredict recipes
- ingot oredict recipes
- stick oredict recipes

### CODE REVIEW

File: src/main/java/gregtech/loaders/load/GT_Loader_MetaTileEntities_Recipes.java [hunk 1]
```diff
@@ -7098,16 +7099,20 @@ private static void run3() {
...
-                Materials.StainlessSteel.getMolten(864L),
-                ItemList.Machine_Multi_Cleanroom.get(1L),
-                1200,
-                120);
+                    GT_Utility.getIntegratedCircuit(1))
+                .itemOutputs(ItemList.Machine_Multi_Cleanroom.get(1L))
+                .fluidInputs(Materials.StainlessSteel.getMolten(864L))
+                .noFluidOutputs()
+                .duration(60 * SECONDS)
+                .eut(120)
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/load/GT_Loader_MetaTileEntities_Recipes.java:7114]: TierEU.RECIPE_MV

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java [hunk 2]
```diff
@@ -26,18 +27,27 @@ public void registerOre(OrePrefixes aPrefix, Materials aMaterial, String aOreDic
...
-                    16);
+                GT_Values.RA.stdBuilder()
+                    .itemInputs(GT_OreDictUnificator.get(OrePrefixes.plate, aMaterial, 1L))
+                    .itemOutputs(
+                        GT_OreDictUnificator.get(OrePrefixes.lens, aMaterial, 1L),
+                        GT_OreDictUnificator.get(OrePrefixes.dustSmall, aMaterial, 1L))
+                    .noFluidInputs()
+                    .noFluidOutputs()
+                    .duration(60 * SECONDS)
+                    .eut(30)
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java:38]: TierEU.RECIPE_LV

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java [hunk 3]
```diff
@@ -50,18 +60,30 @@ public void registerOre(OrePrefixes aPrefix, Materials aMaterial, String aOreDic
...
+                if (GT_OreDictUnificator.get(OrePrefixes.plate, aMaterial, 1L) != null) {
+                    GT_Values.RA.stdBuilder()
+                        .itemInputs(GT_OreDictUnificator.get(OrePrefixes.plate, aMaterial, 1L))
+                        .itemOutputs(
+                            GT_OreDictUnificator.get(OrePrefixes.lens, aMaterial, 1L),
+                            GT_OreDictUnificator.get(OrePrefixes.dustSmall, aMaterial, 1L))
+                        .noFluidInputs()
+                        .noFluidOutputs()
+                        .duration(60 * SECONDS)
+                        .eut(120)
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java:72]: TierEU.RECIPE_MV

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java [hunk 4]
```diff
@@ -50,18 +60,30 @@ public void registerOre(OrePrefixes aPrefix, Materials aMaterial, String aOreDic
...
+                if (GT_OreDictUnificator.get(OrePrefixes.gemExquisite, aMaterial, 1L) != null) {
+                    GT_Values.RA.stdBuilder()
+                        .itemInputs(GT_OreDictUnificator.get(OrePrefixes.gemExquisite, aMaterial, 1L))
+                        .itemOutputs(
+                            GT_OreDictUnificator.get(OrePrefixes.lens, aMaterial, 1L),
+                            GT_OreDictUnificator.get(OrePrefixes.dust, aMaterial, 2L))
+                        .noFluidInputs()
+                        .noFluidOutputs()
+                        .duration(120 * SECONDS)
+                        .eut(30)
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java:84]: TierEU.RECIPE_LV

**Review by boubou19 — CHANGES_REQUESTED** [review]

**Comment by boubou19** [comment]
also you forgot to import `SECONDS` multiple times (see [compilation errors](https://github.com/GTNewHorizons/GT5-Unofficial/actions/runs/4903946223/jobs/8756679365?pr=1957#step:8:667))

**Comment by chochem** [comment]
yea still wip ;)
those are all fixed though

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingIngot.java [hunk 5]
```diff
@@ -44,15 +46,15 @@ public void registerOre(OrePrefixes aPrefix, Materials aMaterial, String aOreDic
...
-                            calculateRecipeEU(aMaterial, 8));
-                    }
 
+                    GT_Values.RA.stdBuilder()
+                        .itemInputs(ItemList.Shape_Mold_Ingot.get(0L))
+                        .itemOutputs(GT_OreDictUnificator.get(OrePrefixes.ingot, aMaterial, 1L))
+                        .fluidInputs(aMaterial.getMolten(144L))
+                        .noFluidOutputs()
+                        .duration(32 * TICKS)
+                        .eut(calculateRecipeEU(aMaterial, 8))
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/oreprocessing/ProcessingIngot.java:56]: 1 second 12 ticks would be better here

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java [hunk 6]
```diff
@@ -26,18 +27,27 @@ public void registerOre(OrePrefixes aPrefix, Materials aMaterial, String aOreDic
...
+                    .addTo(sLatheRecipes);
+
+                GT_Values.RA.stdBuilder()
+                    .itemInputs(GT_OreDictUnificator.get(OrePrefixes.gemExquisite, aMaterial, 1L))
+                    .itemOutputs(
+                        GT_OreDictUnificator.get(OrePrefixes.lens, aMaterial, 3L),
+                        GT_OreDictUnificator.get(OrePrefixes.dust, aMaterial, 1L))
+                    .noFluidInputs()
+                    .noFluidOutputs()
+                    .duration(120 * SECONDS)
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java:48]: 2 minutes would be better

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java [hunk 7]
```diff
@@ -50,18 +60,30 @@ public void registerOre(OrePrefixes aPrefix, Materials aMaterial, String aOreDic
...
-                    30);
+                if (GT_OreDictUnificator.get(OrePrefixes.plate, aMaterial, 1L) != null) {
+                    GT_Values.RA.stdBuilder()
+                        .itemInputs(GT_OreDictUnificator.get(OrePrefixes.plate, aMaterial, 1L))
+                        .itemOutputs(
+                            GT_OreDictUnificator.get(OrePrefixes.lens, aMaterial, 1L),
+                            GT_OreDictUnificator.get(OrePrefixes.dustSmall, aMaterial, 1L))
+                        .noFluidInputs()
+                        .noFluidOutputs()
+                        .duration(60 * SECONDS)
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java:71]: 1 minute

File: src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java [hunk 8]
```diff
@@ -50,18 +60,30 @@ public void registerOre(OrePrefixes aPrefix, Materials aMaterial, String aOreDic
...
+                }
+                if (GT_OreDictUnificator.get(OrePrefixes.gemExquisite, aMaterial, 1L) != null) {
+                    GT_Values.RA.stdBuilder()
+                        .itemInputs(GT_OreDictUnificator.get(OrePrefixes.gemExquisite, aMaterial, 1L))
+                        .itemOutputs(
+                            GT_OreDictUnificator.get(OrePrefixes.lens, aMaterial, 1L),
+                            GT_OreDictUnificator.get(OrePrefixes.dust, aMaterial, 2L))
+                        .noFluidInputs()
+                        .noFluidOutputs()
+                        .duration(120 * SECONDS)
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/oreprocessing/ProcessingLens.java:83]: 2 minutes

**Review by boubou19 — CHANGES_REQUESTED** [review]

**Review by boubou19 — APPROVED** [review]

**Pull request merged**